### ISSUE TITLE
mail with download link arrives in Spam folder

### ISSUE THREAD

**Issue by melvinwevers** [UUDigitalHumanitieslab/texcavator#138]
This used to not be the case.  Can this be fixed?

**Comment by mhkuu** [comment]
Whoops, still have to check whether this actually resolves anything (commit message automatically closed the issue).

**Comment by jgonggrijp** [comment]
I tried downloading something and found the following reasons for the email being marked as spam in the email headers:

    X-Spam-Score: 9.78 (*********) [Tag at 6.00] RDNS_NONE:1.274,TVD_SPACE_RATIO:0.001,URI_ONLY_MSGID_MALF:2.002,SPF(fail:6),DKIM(none:0),207(1),CC(NL:-0.5)
    Received-Spf: fail (filter2-ams.mf.surf.net: domain of ...@uu.nl does not designate 145.100.58.43 as permitted sender) receiver=filter2-ams.mf.surf.net; client-ip=145.100.58.43; envelope-from=<...@uu.nl>; helo=texcavator.surfsaralabs.nl; identity=mailfrom

This looks like the problem is that we are "spoofing" the ...@uu.nl address by simply sending directly from our own SMTP server instead of by authenticating with the UU Exchange server. This causes the SPF failure and the IP mismatch. Did we previously use a different sender address? Otherwise, the UU exchange server probably changed its policies.

Possible fixes:
 - using a different sender address;
 - authenticating with the UU Exchange server (though that would mean all download emails will end up in our sent items folder).

**Comment by mhkuu** [comment]
You're right, thanks! I'll add a setting for the from address so we can set this to 'noreply@surfsaralabs.nl'. Probably best to add 'digitalhumanities@uu.nl' as a reply-to.

**Comment by jgonggrijp** [comment]
Maybe noreply whirlpool texcavator spot surfsaralabs spot nl. Using the surfsaralabs domain directly might piss off the people at SurfSara.

**Comment by mhkuu** [comment]
Should be fixed when we deploy f2c5e6fd451dd9174d47a1adf6f2b9b534d8ba11.

**Comment by mhkuu** [comment]
Unfortunately the e-mail seems to be still marked as spam. We should probably contact SURFsara for a proper solution.
